docs/guides/basic-rbac.mdx

```diff
@@ -1,44 +1,44 @@
 ---
-title: Implement basic Role Based Access Control with Metadata
-description: Learn how to leverage Clerk's publicMetadata to implement your own basic Role Based Access Controls
+title: Implement basic Role Based Access Control with metadata
+description: Learn how to leverage Clerk's publicMetadata to implement your own basic Role Based Access Controls.
 ---
 
-# Implement basic Role Based Access Control (RBAC) with Metadata
+# Implement basic Role Based Access Control (RBAC) with metadata 
 
-Clerk provides [User Metadata](/docs/users/metadata#user-metadata), which is a tool that can be leveraged to build flexible custom logic into your application. This guide will cover using Public Metadata to build a based Role Based Access Control (RBAC) system for your application. 
+To control which users can access certain parts of your application, you can leverage Clerk's [roles](/docs/organizations/roles-permissions#roles) feature. Although Clerk offers roles as part of the feature set for [organizations](/docs/organizations/overview), enabling organizations is *not* required to use roles. This guide will cover setting up a basic Role Based Access Control (RBAC) system for products that don't need organizations.
 
-<Callout type="info">
-Clerk offers Roles as part of the feature set for [Organizations](/docs/organizations/overview).  This guide will cover setting up a basic RBAC system for products that don't need Organizations.
-</Callout>
+This guide assumes that you are using Next.js App Router. The concepts here can be adapted to Next.js Pages Router and Remix.
 
-This guide assumes that you are using Next App Router. The concepts here can be adapted to Next Pages Router and Remix.
+## Configure the session token
 
-## Configuring the Session
+Clerk provides [user metadata](/docs/users/metadata#user-metadata), which is a tool that can be leveraged to build flexible custom logic into your application. `unsafeMetadata` can be read and updated in the browser, and as the name states, it is unsafe to store data in. `privateMetadata` can not be read in the browser. `publicMetadata` can be read by the browser and can only be updated server-side or in the Clerk Dashboard, making it the safest and best choice for this use case.
```

Comment:
We could even expand on the *why* of why we need the metadata. something like:

```suggestion
Clerk provides [user metadata](/docs/users/metadata#user-metadata), which is a tool that can be leveraged to build flexible custom logic into your application. Metadata can be used to store information, and in this case, it can be used to store a user's role.

`unsafeMetadata` can be read and updated in the browser, and as the name states, it is unsafe to store data in. `privateMetadata` can not be read in the browser. `publicMetadata` can be read by the browser and can only be updated server-side or in the Clerk Dashboard, making it the safest and best choice for this use case.
```